src/app/datagrid/datagrid-code-block-editor.demo.ts

```diff
@@ -188,7 +198,7 @@ export class CodeBlockEditorComponent  implements SohoDataGridCellEditor {
   }
 
   focus() {
-    this.codeblock.focus();
+    this.codeblock.focus(event);
```

Comment:
Not seeing the incoming event parameter on the method

Reply:
@deep7102 Something like this >
focus(event: JQuery.TriggeredEvent) {

Reply:
Hi @fitzorama I did tried that did not work
- Add in to interface here `/projects/ids-enterprise-ng/src/lib/datagrid/soho-datagrid.d.ts` as
`focus(event: JQuery.TriggeredEvent) {`
- Then use here `/src/app/code-block/code-block.component.ts` as 
`focus(event: JQuery.TriggeredEvent) {`
- See it always undefined